buildsystem/templates/Doxyfile.in

```diff
@@ -47,7 +47,7 @@ PROJECT_BRIEF          =
 # exceed 55 pixels and the maximum width should not exceed 200 pixels.
 # Doxygen will copy the logo to the output directory.
 
-PROJECT_LOGO           =
+PROJECT_LOGO           = "@CMAKE_SOURCE_DIR@/assets/logo/banner_doxygen.png"
```

Comment:
does it have to be a new image? Or can we reuse the `banner.png`? I assume with a proper css we can avoid the resized redundant image.